steps/bitrise-step-appdome-build-2secure-android/1.0.2/step.yml

```diff
@@ -0,0 +1,52 @@
+title: Appdome-Build-2Secure step for Android
+summary: |
+  Builds a mobile app using Appdome's platform
+description: |
+  Integration that allows activating security and app protection features, bulding and signing mobile apps using Appdome's API. For detailes see: https://www.appdome.com/how-to/appsec-release-orchestration/mobile-appsec-cicd/use-appdome-build-2secure-step-for-bitrise
+website: https://github.com/Appdome/bitrise_build-2secure-android
```

Comment:
Is this the correct URL? I noticed the git source url goes to a different repository below.
`https://github.com/Appdome/bitrise-step-appdome-build-2secure-android.git`
vs
`https://github.com/Appdome/bitrise_build-2secure-android`

Reply:
That's correct.